lib/mysql.js

```diff
@@ -453,10 +459,6 @@ MySQL.prototype.fromColumnValue = function(prop, val) {
         }
         break;
       default:
-        if (!Array.isArray(prop.type) && !prop.type.modelName) {
-          // Do not convert array and model types
```

Comment:
If you remove this `default` case, does it mean that we are no longer going to convert other types? What are the implications for existing applications, can we break any edge cases?

Reply:
@bajtos The fix here is tricky, I have spent lots of time to figure out what is the expected behaviour across all the connectors:

When running tests in juggler v4, the type conversion here causes https://github.com/strongloop/loopback-datasource-juggler/blob/master/test/datatype.test.js#L193-L199 fail. This test expects the connector returns the value from db AS IT IS.

And if you look at the postgresql connector https://github.com/strongloop/loopback-connector-postgresql/blob/master/lib/postgresql.js#L246, it also returns the value WITHOUT conversion.

Therefore I believe at least for juggler v4, the expected behaviour is connectors don't do type conversions for unknown types.

Reply:
I see, thank you for explaining the details to me.